ktor-client/ktor-client-core/js/src/io/ktor/client/fetch/LibDom.kt

```diff
@@ -1,392 +0,0 @@
-/*
- * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
- */
-
-package io.ktor.client.fetch
-
-import kotlin.js.Promise
-
-// external fun fetch(input: Request, init: RequestInit? = definedExternally): Promise<Response>
-
-public external fun fetch(input: String, init: RequestInit? = definedExternally): Promise<Response>
```

Comment:
We can't delete declarations now, only in 2.0.0. Can you keep it, but add deprecations? Then it's ok to go to the master branch.